releases/nightly-build/compose-files/docker-compose-nexus-arm64.yml

```
@@ -0,0 +1,560 @@
+# /*******************************************************************************
+#  * Copyright 2018 Dell Inc.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  * @authormaster
+#  * EdgeX Foundry, Fuji, version master
+#  * added: Jun 30, 2019
+#  *******************************************************************************/
+
+version: '3.4'
+
+# all common shared environment variables defined here:
+x-common-env-variables: &common-variables
+  edgex_registry: consul://edgex-core-consul:8500
+  Clients_CoreData_Host: edgex-core-data
+  Clients_Logging_Host: edgex-support-logging
+  Logging_EnableRemote: "true"
+
+volumes:
+  db-data:
+  log-data:
+  consul-config:
+  consul-data:
+  consul-scripts:
+  portainer_data:
+  vault-init:
+  vault-config:
+  vault-file:
+  vault-logs:
+  secrets-setup-cache:
+
+services:
+  volume:
+    image: nexus3.edgexfoundry.org:10004/docker-edgex-volume-arm64:master
+# consul:1.3.1
+    container_name: edgex-files
+    networks:
+      - edgex-network
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+
+  consul:
+    image: nexus3.edgexfoundry.org:10001/arm64v8/consul:1.3.1
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+    container_name: edgex-core-consul
+    hostname: edgex-core-consul
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-consul
+    volumes:
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - consul-scripts:/consul/scripts
+      - vault-config:/vault/config
+      - /tmp/edgex/secrets/ca:/tmp/edgex/secrets/ca:ro
+      - /tmp/edgex/secrets/edgex-vault:/tmp/edgex/secrets/edgex-vault:ro
+      - /tmp/edgex/secrets/edgex-kong:/tmp/edgex/secrets/edgex-kong:ro
+    depends_on:
+      - volume
+
+  config-seed:
+    image: nexus3.edgexfoundry.org:10004/docker-core-config-seed-go-arm64:master
+    container_name: edgex-config-seed
+    hostname: edgex-core-config-seed
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-config-seed
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+      - consul
+
+  vault:
+    image: nexus3.edgexfoundry.org:10004/docker-edgex-secret-store-go-arm64:master
```

this should be from HashiCorp's official image so that we can upgrade to different version without rebuild the image.  See issue https://github.com/edgexfoundry/edgex-go/issues/1716 resolved
```suggestion
    image: vault:1.0.3
```

---

resolved